tests/data/results_npz/splinempe/run00136766.evt000007637140.neutrino_1.json

```diff
@@ -0,0 +1,93 @@
+{
+    "nside-1": {
+        "columns": [
+            "index",
+            "llh",
+            "E_in",
+            "E_tot"
+        ],
+        "data": [
+            [
+                0,
+                1899.893825371186,
+                NaN,
+                NaN
```

Comment:
Does the CI test read this JSON file? There looks to be a similarly named npz file. That said, I do like the json format more for tests since it's human-readable

Reply:
> Does the CI test read this JSON file? There looks to be a similarly named npz file. That said, I do like the json format more for tests since it's human-readable

Not yet, but adding the JSON goes in that direction. My idea was to change the CI test to read JSON for all three recos once SplineMPE is merged.

I suppose `NaN` values can be handled gracefully even if they are not standard JSON. If not, I guess we can output zeros instead?

Reply:
OK, I am taking care of this non-destructively in this PR.